frontend/device/android/powerd.lua

```diff
@@ -3,17 +3,48 @@ local _, android = pcall(require, "android")
 
 local AndroidPowerD = BasePowerD:new{
     fl_min = 0, fl_max = 25,
```

Comment:
probably these values can be retrieved from the backend?

Reply:
I think no. These are just for scaling the backend value to 0-25 in the frontend (gestures report).